src/components/Store.tsx

```diff
@@ -145,8 +146,9 @@ let filterOptions = {
 export type FilterOption = keyof typeof filterOptions
 export const FILTER_OPTIONS = Object.keys(filterOptions) as FilterOption[]
 
-export function Store({ character, sortOption, filterOption }: { character?: Character, sortOption: SortOption, filterOption: FilterOption }) {
-	let store = useStore(character)
+export function Store({ character, storeType, sortOption, filterOption }: { character: Character|undefined, storeType: StoreType, sortOption: SortOption, filterOption: FilterOption }) {
```

Comment:
nitpick: For readability purposes, there should probably be a space on either side of the pipe symbol between Character and undefined
Also, should the log below this line be removed?

Reply:
Ah, good spot on the log line. Have added spaces to bring it into line with the rest of the code too.